### DISCUSSION TITLE
群内发送的url经常从中途断开，短链配置也没生效

### DISCUSSION THREAD

**Discussion by aimiku** [Tsuk1ko/CQ-picfinder-robot#180] [问题求助] [answered]
![image](https://user-images.githubusercontent.com/10482479/118457280-2f017200-b72c-11eb-97ca-f42cf4710e64.png)
![image](https://user-images.githubusercontent.com/10482479/118457316-36c11680-b72c-11eb-9bcc-eb58d100bfeb.png)

**Comment by Tsuk1ko** [comment, marked as answer]
先换 oy.mk 吧，项目里现存的 t.cn 的 api 好像用不了，晚上我更新下

**Comment by Tsuk1ko** [comment]
更了，is.gd 墙了，t.cn 已经没得用了，我都删了，顺便加了我自己的 yww.uy

**Reply by aimiku** [reply]
oy.mk 好像也烂了，会提示该网站存在风险
